src/Operators/derivative_operators.jl

```diff
@@ -14,9 +14,13 @@
 
 @inline ∂xᶜᵃᵃ(i, j, k, grid::ARG, f::F, args...) where F<:Function = δxᶜᵃᵃ(i, j, k, grid, f, args...) / Δx(i, j, k, grid)
 @inline ∂xᶠᵃᵃ(i, j, k, grid::ARG, f::F, args...) where F<:Function = δxᶠᵃᵃ(i, j, k, grid, f, args...) / Δx(i, j, k, grid)
+### @inline ∂xᶠᵃᵃ(i, j, k, grid::Union{ARG, AHCG}, f::F, args...) where F<:Function = δxᶠᵃᵃ(i, j, k, grid, f, args...) / Δxᶠᵃᵃ(i, j, k, grid)
+@inline ∂xᶠᵃᵃ(i, j, k, grid::Union{ARG, AHCG}, c) = δxᶠᵃᵃ(i, j, k, grid, c) / Δxᶠᵃᵃ(i, j, k, grid)
```

Comment:
This function doesn't work for a lat-lon grid --- are these lines added by mistake?